src/System.Private.CoreLib/shared/System/Globalization/JapaneseCalendar.WinRT.cs

```diff
@@ -54,9 +55,9 @@ private static string GetJapaneseEnglishEraName(int era)
             return era <= s_JapaneseErasEnglishNames.Length ? s_JapaneseErasEnglishNames[era - 1] : " ";
         }
 
-        private static bool GetJapaneseEraInfo(int era, out DateTimeOffset dateOffset, out string eraName, out string abbreviatedEraName)
+        private static bool GetJapaneseEraInfo(int era, out DateTimeOffset dateOffset, out string? eraName, out string? abbreviatedEraName)
```

Comment:
I'm not 100% sure about this ones.

@tarekgh do you know if these out string parameters can be null?

Reply:
The method will return false if we couldn't get non null strings.

Reply:
@krwq what happen if one of these strings is null?

Reply:
https://github.com/dotnet/coreclr/blob/e783bbeb616f77279803cdc51c83f97e0b9ab23e/src/System.Private.CoreLib/shared/System/Globalization/JapaneseCalendar.WinRT.cs#L32-L35

assuming we return false on failure we are good - if API returns true AND null as one of the strings then it will be populated to EraInfo constructor

Reply:
Then those 2 values in the call site should be nullable. I guess that could break the corert build

https://github.com/dotnet/coreclr/blob/e783bbeb616f77279803cdc51c83f97e0b9ab23e/src/System.Private.CoreLib/shared/System/Globalization/JapaneseCalendar.WinRT.cs#L29

Reply:
interesting no compile errors - will file issue on that later

Reply:
cc: @jaredpar

Reply:
actually this is my bad - forgot the private CI doesn't build core rt

Reply:
Oh I thought you actually built corert with this changes locally. Might be worth doing it just to make sure.

Reply:
can we add it to CI?

Reply:
It is not trivial. I don’t think it is worth spending a day on adding it. If we see it starts causing pain then we can invest in it.

Reply:
im not sure how to build corert locally as well - we can fix any errors on the main PR since we have CI in there - there is very little CoreRT changes